src/models/ControlMap.ts

```diff
@@ -20,11 +20,27 @@ export class ControlMap {
         this.width = width;
         this.height = height;
 
+        let cleanUpTransform: boolean;
```

Comment:
should cleanUpTransform be initialized to false?

Reply:
yes, that's probably neater (even though it's implicitly set to `false`. I'll fix shortly